### DISCUSSION TITLE
TTS - ElevenLabs default settings

### DISCUSSION THREAD

**Discussion by cvrle77** [SubtitleEdit/subtitleedit#8318] [Ideas]
Please, do change voice model you're using to eleven_multilingual_v2, it is their best model and most versatile.
You're using their oldest and worst model.
https://help.elevenlabs.io/hc/en-us/articles/17883183930129-What-models-do-you-offer-and-what-is-the-difference-between-them

Piece of code I am referring to:
/src/ui/Forms/Tts/TextToSpeech.cs

`1034                var data = "{ \"text\": \"" + Json.EncodeJsonText(text) + "\", \"model_id\": \"eleven_monolingual_v1\", \"voice_settings\": { \"stability\": 0.5, \"similarity_boost\": 0.5 } }";`

If possible, give us option to set stability and similarity, if not, similarity should be at 1, instead of 0.5 and stability somewhere around 0.8
These settings are important if you have professional voice clone generated, not for voice library (which I happen to have)

I've changed these for myself, and recompiled SE, to get my voice as it appears on their website to be 100% match. 

Also, please add eleven-labs-voices.json download button, so SE can use API to download fresh one, when we need it.

**Comment by niksedk** [comment]
Model + settings updated: https://github.com/SubtitleEdit/subtitleedit/releases/download/4.0.5/SubtitleEditBeta.zip

Also, the voices combo-box has a context menu now:
![image](https://github.com/SubtitleEdit/subtitleedit/assets/1261331/57d69d83-ec3a-485a-91e8-e509281681c0)

**Comment by cvrle77** [comment]
Flawless. 

I do have another request, if possible: 
please add Force Stereo checkbox so we don't have to use other tools to convert mono to stereo output (ffmpeg -i input.wav -ac 2 output.wav) if possible.

**Comment by cvrle77** [comment]
Oh, and PLEASE give us a choice to save working folder which will be used as temporary and final output folder. 
It's next to impossible to remember current location and random folder naming. ATM it's a mess.

![image](https://github.com/SubtitleEdit/subtitleedit/assets/16167396/e109f19d-0951-4f9f-b855-32ca7b8d5c67)
